macros/edr/data_monitoring/monitors/column_boolean_monitors.sql

```diff
@@ -1,7 +1,7 @@
 {% macro count_true(column_name) -%}
-    coalesce(sum(case when cast({{ column_name }} as {{ elementary.edr_type_bool() }}) is true then 1 else 0 end), 0)
+    coalesce(sum(case when cast({{ column_name }} as {{ elementary.edr_type_bool() }}) = true then 1 else 0 end), 0)
```

Comment:
The "is true" syntax seems to work only for BQ, where = seems to always work and I think is good enough